### PR TITLE
fix: make the vue example work

### PR DESCRIPTION
**Pull request by sodatea**

### CODE REVIEW

File: examples/vue/package.json [hunk 1]
```diff
@@ -12,15 +12,13 @@
     "vue": "^3.0.5"
   },
   "devDependencies": {
-    "@testing-library/vue": "^5.6.1",
+    "@testing-library/vue": "^6.4.0",
```
Comment (sodatea) [inline examples/vue/package.json:15]: Vue 3 requires `@testing-library/vue` v6

File: examples/vue/package.json [hunk 2]
```diff
@@ -12,15 +12,13 @@
...
     "@vitejs/plugin-vue": "^1.1.5",
+    "@vitejs/plugin-vue-jsx": "^1.1.2",
```
Comment (sodatea) [inline examples/vue/package.json:17]: In case anyone needs it

File: examples/vue/package.json [hunk 3]
```diff
@@ -12,15 +12,13 @@
...
     "@vue/compiler-sfc": "^3.0.5",
     "@web/test-runner": "^0.12.17",
     "chai": "^4.3.4",
     "vite": "^2.1.3",
-    "vite-web-test-runner-plugin": "file:../..",
-    "vue-router": "^3.5.1",
```
Comment (sodatea) [inline examples/vue/package.json:22]: `vue-router` & `vuex` are not required in this example.

File: examples/vue/package.json [hunk 4]
```diff
@@ -12,15 +12,13 @@
...
-    "vue-template-compiler": "^2.6.12",
```
Comment (sodatea) [inline examples/vue/package.json:23]: `vue-template-compiler` is a Vue 2 dependency.

File: examples/vue/src/App.test.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ import App from './App.vue';
 
 describe('<App>', () => {
   it('renders vue documentation link', () => {
-    const { getByText } = render(<App />);
+    const { getByText } = render(App);
```
Comment (sodatea) [inline examples/vue/src/App.test.js:7]: You don't need JSX syntax to render the component with `@testing-library/vue`.

File: examples/vue/web-test-runner.config.js [hunk 6]
```diff
@@ -3,4 +3,19 @@ module.exports = {
   plugins: [
     vite(),
   ],
+  coverageConfig: {
+    include: ['src/**']
```
Comment (sodatea) [inline examples/vue/web-test-runner.config.js:7]: Errors will be thrown if `@web/test-runner` tries to generate coverage report for files under `node_modules/.vite/`.
So here we use a whitelist to only include the source files.

**Comment by betaboon** [comment]
heya. very cool.

a couple questions tho:
- could you please combine the changes that make the vue-example working into a single commit
- could you please separate the coverageConfig stuff into a seperate commit and add it for the other examples as well (including a `--coverage` in the package.json->scripts->test)

**Comment by sodatea** [comment]
Sure, done!

**Comment by betaboon** [comment]
awesome. thank you very much :)

**Pull request merged**